config/main.py

```diff
@@ -14,34 +14,45 @@
 import ipaddress
 from swsssdk import ConfigDBConnector, SonicV2Connector, SonicDBConfig
 from minigraph import parse_device_desc_xml
+from config_mgmt import ConfigMgmtDPB
 from utilities_common.intf_filter import parse_interface_in_filter
 
 import aaa
 import mlnx
 import nat
 
+# import port config file path
+from sfputil.main import get_platform_and_hwsku
```

Comment:
Prefer not to import from another utility. All common functions should be imported from a library. Please import this function from here instead: https://github.com/Azure/sonic-utilities/blob/master/utilities_common/util_base.py

Reply:
Modified